hadoop-ozone/dist/src/shell/ozone/stop-ozone.sh

```diff
@@ -59,7 +59,9 @@ ozone_uservar_su ozone datanode "${OZONE_HOME}/bin/ozone" \
 
 #---------------------------------------------------------
 # Ozone Manager nodes
-OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf ozonemanagers 2>/dev/null)
+TMP_OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf ozonemanagers 2>/dev/null)
+OM_NODES=`echo $TMP_OM_NODES | grep -o '\([0-9]\{1,3\}\.\)\{3\}[0-9]\{1,3\}' | sort -d | uniq`
+
```

Comment:
This change also seems to be unrelated to Ranger.

Reply:
I am very sorry that I did not read it carefully when submitting it